demo/runners/support/agent.py

```diff
@@ -510,6 +524,11 @@ async def handle_webhook(self, topic: str, payload, headers: dict):
             wallet_id = headers.get("x-wallet-id")
             method = getattr(self, handler, None)
             if method:
+                self.log(
+                    f"Agent called controller webhook: POST {self.webhook_url}",
+                    (f"\nFor wallet: {wallet_id}" if wallet_id else ""),
+                    (f"\nPayload: {repr_json(payload)}" if payload else repr_json({})),
+                )
                 EVENT_LOGGER.debug(
```

Comment:
Web hook will be logged twice if run with `--events`

Reply:
Now only relying on event_logger for demonstration.
![image](https://user-images.githubusercontent.com/9292265/104378937-950b8d80-54dd-11eb-9f16-8e68731a9e31.png)